mautic/mautic
# Company merge event

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This is follow up of https://github.com/mautic/mautic/pull/11228
Bring also some improvements during redirect after merge to detail view of company




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Try merge company
3. Check If redirection after success merge going to detail view of company instead of edit view




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11748"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
